install.ps1

```diff
@@ -0,0 +1,39 @@
+
+# Enable TLS 1.2 since it is required for connections to GitHub
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+# Determine latest Deno release via GitHub API
+$LatestRelease = Invoke-WebRequest `
+  -Uri "https://api.github.com/repos/denoland/deno/releases/latest" `
+    | ConvertFrom-Json `
+    | Select-Object -ExpandProperty "tag_name"
+
+# Create ~\.deno\bin directory if it doesn't already exist
+$DenoBin = "${Home}\.deno\bin"
+if (!(Test-Path $DenoBin)) {
+  New-Item -Path $DenoBin -ItemType "Directory" > $null
+}
+
+# Download latest Deno release .zip file
+$DenoZip = "${DenoBin}\deno_win_x64.zip"
+Invoke-WebRequest `
+  -Uri "https://github.com/denoland/deno/releases/download/${LatestRelease}/deno_win_x64.zip" `
+  -OutFile $DenoZip
+
+# Extract .zip file
+Expand-Archive `
+  -Path $DenoZip `
+  -DestinationPath $DenoBin `
+  -Force # Using -Force to overwrite deno.exe if it already exists
+
+# Remove .zip file
+Remove-Item -Path $DenoZip
+
+# Get Path environment variable for the current user
+$User = [EnvironmentVariableTarget]::User
+$Path = [Environment]::GetEnvironmentVariable("Path", $User)
+
+# Add Deno to the Path if it hasn't been added already
+if (!$Path.Contains("${DenoBin};") -And !$Path.EndsWith($DenoBin)) {
```

Comment:
> Seems a little duplicative; just use `$Path -split ";" -contains $DenoBin`

Agreed.

Actually, there's another edge case here. Both `C:\bin` and `C:\bin\` (with a trailing `\`) work. So I think I'm gonna go for:

```powershell
$Paths = $Path -split ";"
$IsInPath = (
  $Paths -contains $DenoBin -or
  $Paths -contains "${DenoBin}\"
)

if (!IsInPath) {
  # ...
}
```

> It'd be nice to ask users before updating their path (or at least tell them).

Agreed. I'm gonna add a console log for that. And I think the documentation in the README.md should contain a note that this will update the Path.

> It'd be nice to update $env:PATH too so they can immediately start using deno.

True! I forgot about that.

Reply:
> Actually, there's another edge case here. Both C:\bin and C:\bin\ (with a trailing \) work. So I think I'm gonna go for:

If you want to cover edge cases, I know some more: a path entry may be wrapped in or contain " quote marks. 
We could do: `($Path -split ";" | Get-Item -ErrorAction Ignore).FullPath -contains $DenoBin` but I'm sure that still leaves other edge cases...

So just do whatever, it's v0.1 anyway.